src/audio/copier/copier.c

```diff
@@ -962,6 +962,7 @@ static int copier_comp_trigger(struct comp_dev *dev, int cmd)
 	struct comp_dev *dai_copier;
 	struct copier_data *dai_cd;
 	struct comp_buffer *buffer;
+	struct comp_buffer __sparse_cache *buffer_c;
```

Comment:
I prefer to move this definition after "struct copier_data *cd = comp_get_drvdata(dev);"